src/ContextMenuTrigger.js

```diff
@@ -49,26 +53,37 @@ export default class ContextMenuTrigger extends Component {
         }
     }
 
+
     handleTouchEnd = (event) => {
         event.preventDefault();
         clearTimeout(this.touchstartTimeoutId)
     }
 
+    handleMouseMove = (event)=>{
+        event.persist()
+        this.mouseMoveEvent = event
```

Comment:
events in React are synthetic events and are reused.
I believe this will not work as expected.

why not use this instead?
```js
 this.mousePosition = {
    x:event.pageX,
    y:event.pageY
}
```